pkg/kubelet/kubelet.go

```diff
@@ -2346,10 +2348,13 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 	// Check whether network is configured properly
 	networkConfigured := kl.doneNetworkConfigure()
 
+	selinuxRequired := capabilities.Get().EnableSELinuxIntegration
+	selinuxConfigured := (selinuxRequired && selinux.SelinuxEnabled()) || !selinuxRequired
```

Comment:
Unfortunately the libcontainer selinux library doesn't provide a stub for !linux, so you have to abstract this behind a golang build flag.  See `root_context_{linux,unsupported}.go`